samples/samples-js/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "samples-js",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "start": "func start",
+    "test": "echo \"No tests yet...\""
```

Comment:
What should this really be instead of the echo command?

Reply:
Will have to look up ways of adding tests to the node js app and add that here. I'll be working on it.

Reply:
Just remove it then. No reason to keep stuff around if it's not doing anything.

Reply:
@MaddyDev This seems to have snuck back in again...

Reply:
I added it back since one test started failing once I removed it. Going to look into that but don't want to block the PR on it.